### DISCUSSION TITLE
[docker] ERROR: gremlin-server/gremlin-server.yaml is unreadable after any change

### DISCUSSION THREAD

**Discussion by Kukant** [JanusGraph/janusgraph#3355] [Q&A]
Hi all,
I am testing Janusgraph using docker. I just run the server --detach(ed) and gremlin console directly. While loading data into the graph, I encountered timeout so I decided to edit `conf/janusgraph-server.yaml` evaluationTimeout. However, when I save the file and restart the server, I am getting this error message:

```
root@01e9cf3d39fa:/opt/janusgraph# ./bin/janusgraph-server.sh start
ERROR: gremlin-server/gremlin-server.yaml is unreadable
```

I then changed the timeout value to previous value and it still did not work. I tried to debug it using these steps:
1. made a copy of the original config file to `conf/janusgraph-server.original.yaml`
2. made my small change to the timeout in  `conf/janusgraph-server.yaml`
3. made a diff, it showed what I expected, only change was the evaluationTimeout value.
4. got the error again.
5. reset the evaluationTimeout back to previous value.
6. diff again -> no difference found
7. got the error again :(. 

Could you please help me with that? Am I using it wrong?

Also, I saw the note in the beginning of the file:
```
# NOTE: THIS FILE IS GENERATED VIA "update.sh"
# DO NOT EDIT IT DIRECTLY; CHANGES WILL BE OVERWRITTEN.
```
I could not find any script of this name.

**Comment by vtslab** [comment]
Sounds weird indeed. If you edit the yaml file in MS Windows be sure that the Linux line endings stay in place (e.g. use notepad++).

The remark about update.sh does not make sense to me either. The changes are copied into the image during building (assuming you cloned the janusgraph-docker repo and build the image locally and run the local image).
